### PR TITLE
fix(FEC-8717): the dropdown is cut on large amount of options

### PR DESCRIPTION
**Pull request by odedhutzler**
adding a third option to the menu position.
1. Limit the menu height to 80% of the player height. (`0.8*playerHeight`)
2. Position the menu above the bottom of the player.
3. When it's the lowest menu (e.g. captions) - it is indeed almost at the bottom. if its the upper menu (audio) then it will be more up because the menu position is absolute to the `Dropdown` component.

Before:
screen shot 2019-01-29 at 13 54 37

After: 
screen shot 2019-01-29 at 12 47 56

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

**Comment by kaltura-hooks** [comment]
Hi @odedhutzler,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

File: src/components/menu/menu.js [hunk 1]
```diff
@@ -88,10 +88,17 @@ class Menu extends Component {
     const playerContainerRect = this.props.playerClientRect;
     const offsetBottom = playerContainerRect.bottom - menuElementRect.bottom;
     const height = offsetBottom + menuElementRect.height;
-    if (height > playerContainerRect.height) {
+    if (height < playerContainerRect.height) {
```
Comment (yairans) [inline src/components/menu/menu.js]:
```suggestion
    if (height <= playerContainerRect.height) {
```

File: src/styles/_dropdown.scss [hunk 2]
```diff
@@ -72,6 +72,10 @@
   &.left {
     right: 0;
   }
+  &.stick-bottom{
+    top: -60px;
```
Comment (yairans) [inline src/styles/_dropdown.scss]:
```suggestion
    bottom: -60px;
```

File: src/components/menu/menu.js [hunk 3]
```diff
@@ -86,12 +86,15 @@ class Menu extends Component {
   getPosition(): Array<string> {
     const menuElementRect = this._menuElement.getBoundingClientRect();
     const playerContainerRect = this.props.playerClientRect;
-    const offsetBottom = playerContainerRect.bottom - menuElementRect.bottom;
-    const height = offsetBottom + menuElementRect.height;
-    if (height > playerContainerRect.height) {
+
+    if (menuElementRect.top >= playerContainerRect.top) {
+      return [style.top, style.left];
+    } else if (menuElementRect.bottom + menuElementRect.height < playerContainerRect.bottom) {
```
Comment (yairans) [inline src/components/menu/menu.js]:
```suggestion
    } else if ((menuElementRect.bottom + menuElementRect.height) < playerContainerRect.bottom) {
```

Comment (yairans) [reply src/components/menu/menu.js]: add comments

**Pull request merged**